### PR TITLE
dbt: add page

### PR DESCRIPTION
**Pull request by cobac**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by gutjuri — APPROVED** [review]

**Comment by Waples** [comment]
lgtm but i have not tested the actuall commands. Peer review requested.

File: pages/common/dbt.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+`dbt run`
+
+- Run all tests of `example_model`:
+
+`dbt test --select example_model`
+
+- Build (load seeds, run models, snapshots, and tests associated with) `example_model` and its downstream dependents:
+
+`dbt build --select example_model+`
```
Comment (spageektti) [inline pages/common/dbt.md:20]:
```suggestion
`dbt build --select example_model`
```
Is the `+` a typo?

**Review by spageektti — APPROVED** [review]
Hi. Thanks for your contribution.

File: pages/common/dbt.md (hunk 1 above)
Comment (cobac) [reply pages/common/dbt.md:20]: No, it's the syntax for  selecting for all downstream dependents of `example_model`.

**Pull request merged**